acme_diags/driver/enso_diags_driver.py

```diff
@@ -54,7 +54,7 @@ def calculate_nino_index_model(data, nino_region_str, parameter):
     """
 
     try:
-        sst = data.get_timeseries_variable('SST')
+        sst = data.get_timeseries_variable('TS')
```

Comment:
Is this change ok? If there are any islands in the `nino_region` wouldn't `TS` then also include land surface temperatures along with the sea surface temperatures?

Reply:
This change is okay. Good point about the island, I visually checked all the `nino_regions`, I think use `TS` in this case is a pretty safe assumption.